x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -214,90 +175,73 @@ export const ActionTypeForm = ({
     <>
       {actionGroups && selectedActionGroup && setActionGroupIdByIndex && (
         <>
-          <EuiFlexGroup component="div">
-            <EuiFlexItem grow={true}>
-              <EuiFormControlLayout
```

Comment:
Caroline recommended that I move the `prepend` prop from this component straight into the super select and and remove the `EuiFormControlLayout` as it wasn't needed. This was also causing rounded edges which wasn't consistent with the prepend on the combobox we use below

![image](https://user-images.githubusercontent.com/56361221/144661950-0d67bfb4-c7b9-4dd4-9e75-c1aba9d0e5b4.png)